### ISSUE TITLE
Feature to run all test for project

### ISSUE THREAD

**Issue by larebsyed** [jojojames/flycheck-jest#4]
I believe it will be great to have a feature to run all test for a particular project and show its result. In modeline, so when we are changing files we know that test are not broken.

**Comment by jojojames** [comment]
I'll think about it, no timelines though.

Something I want to do before that is to look into the new flymake stuff and potentially port most of my flycheck-* packages to flymake.

**Comment by larebsyed** [comment]
why do we want to move to flymake, isn't flycheck the new de-facto standard for this kind of job?

**Comment by jojojames** [comment]
It's been refactored/made better in emacs 26.

**Comment by larebsyed** [comment]
Hey whats the update in this?

**Comment by jojojames** [comment]
Open to a PR.

**Comment by larebsyed** [comment]
@jojojames Did you explore the flymake? Have you replaced flycheck with flymake for your workflow?

**Comment by jojojames** [comment]
Yes, I'm using flymake where there's a flymake extension. (eglot and the various flymake extensions I've written) @larebsyed
